drivers/storage/portworx/deployment.go

```diff
@@ -559,33 +654,26 @@ func (t *template) getVolumes() []v1.Volume {
 	return volumes
 }
 
-// getImageURN returns the docker image URN depending on a given registryAndRepo and given image
-func getImageURN(registryAndRepo, image string) string {
-	if image == "" {
-		return ""
+func (t *template) getCSIVolumeInfoList() []volumeInfo {
+	registrationVol := volumeInfo{
+		name:         "registration-dir",
+		hostPath:     "/var/lib/kubelet/plugins_registry",
+		hostPathType: hostPathTypePtr(v1.HostPathDirectoryOrCreate),
 	}
-
-	registryAndRepo = strings.TrimRight(registryAndRepo, "/")
-	if registryAndRepo == "" {
-		// no registry/repository specifed, return image
-		return image
+	if t.csiVersions.useOlderPluginsDirAsRegistration {
+		registrationVol.hostPath = "/var/lib/kubelet/plugins"
 	}
 
-	imgParts := strings.Split(image, "/")
-	if len(imgParts) > 1 {
-		// advance imgParts to swallow the common registry
-		if _, present := commonDockerRegistries[imgParts[0]]; present {
-			imgParts = imgParts[1:]
-		}
-	}
-
-	// if we have '/' in the registryAndRepo, return <registry/repository/><only-image>
-	// else (registry only) -- return <registry/><image-with-repository>
-	if strings.Contains(registryAndRepo, "/") && len(imgParts) > 1 {
-		// advance to the last element, skipping image's repository
-		imgParts = imgParts[len(imgParts)-1:]
-	}
-	return registryAndRepo + "/" + path.Join(imgParts...)
+	volumeInfoList := []volumeInfo{registrationVol}
+	volumeInfoList = append(volumeInfoList,
+		volumeInfo{
+			name:         "csi-driver-path",
+			hostPath:     "/var/lib/kubelet/plugins/com.openstorage.pxd",
```

Comment:
@lpabon with the CSI driver name change, all of these paths will need to change too right?

What version is the driver name change effective from?